x/oracle/types/ballot.go

```diff
@@ -173,9 +174,14 @@ func (pb ExchangeRateBallots) Swap(i, j int) {
 
 // ValidatorPerformance keeps track of a validator performance in the voting period.
 type ValidatorPerformance struct {
-	Power        int64 // tendermint consensus power
-	RewardWeight int64 // how much of the rewards this validator should receive, units of consensus power
-	WinCount     int64
+	// Tendermint consensus voting power
+	Power int64
+	// RewardWeight: Weight of rewards the validator should receive in units of
+	// consensus power.
+	RewardWeight int64
+	WinCount     int64 // Number of valid votes for which the validator will be rewarded
+	AbstainCount int64 // Number of abstained votes for which there will be no reward or punishment
```

Comment:
I don't think we're using AbstainCount usefully anywhere. Let's just get rid of it?

Reply:
There's a ticket open to as an event for this